javascript/javascript1/week3/Whenwillbethere.js

```diff
@@ -0,0 +1,24 @@
+
+// When will be there . 
+// Time = distance/ speed
+
+
+  const travelInformation ={
+    speed: 50,  
+     distinationDistance : 432, // km
+     
+
+  }
+
+   function travelTimeCalcu(myTime){
```

Comment:
Please, format the code. 
It should be super easy
```
On Windows Shift + Alt + F
On macOS Shift + Option + F
On Linux Ctrl + Shift + I
```
https://mkyong.com/vscode/how-to-format-source-code-in-visual-studio-code-vscode/